.gitignore

```diff
@@ -72,3 +72,6 @@ venv/
 venv-*/
 out/
 protos/**/*_pb2*
+
+### Intermitten build result
+fallback-control-plane
```

Comment:
Removed

Reply:
I mean I don't mind it here, just was wondering if we could configure somewhere to put the binary to `./out/` - which is already excluded because it's intended for this exact case.